lib/updaters/index.js

```diff
@@ -17,6 +17,12 @@ function getUpdaterByFilename (filename) {
   if (PLAIN_TEXT_BUMP_FILES.includes(filename)) {
     return getUpdaterByType('plain-text')
   }
+  if (/pom.xml$/.test(filename)) {
+    return getUpdaterByType('pom')
+  }
+  if (/build.gradle$/.test(filename)) {
+    return getUpdaterByType('gradle')
+  }
   throw Error(
     `Unsupported file (${filename}) provided for bumping.\n Please specifcy the updater \`type\` or use a custom \`updater\`.`
```

Comment:
Seems like there is a typo.
Let's change `specifcy` to `specify`.

#typo-error

Reply:
Fixed, thanks @akashchandwani!